### ISSUE TITLE
Policy commands default to user accounts, leading to suboptimal UX

### ISSUE THREAD

**Issue by joeduffy** [pulumi/pulumi#14132]
By default, `pulumi policy ls` (and other commands) default to your personal account for the organization. This yields:

```
error: List Policy Packs failed: [400] Bad Request: Policy as code feature not enabled.
```

Given that policies are an organization feature, shouldn't organization by required? Or should we smartly default to your only organization, if you are part of one?

I understand technically we can enable policies on individual accounts, but that is a very rare circumstance.

**Comment by lukehoban** [comment]
> Or should we smartly default to your only organization, if you are part of one?

This was (at least partly) fixed in https://github.com/pulumi/pulumi/pull/14090 I believe via picking up the default org the user has set in their CLI. 

There is also https://github.com/pulumi/pulumi/issues/14033 tracking broader improvements for this.

**Comment by joeduffy** [comment]
Since policy commands are designed for organizations, shouldn't this just be a required argument if someone hasn't set a default organization? (Which I have not, since I bounce between many on a regular basis and prefer to be explicit.)

**Comment by lukehoban** [comment]
Yeah - that seems like a reasonable alternative to what’s suggested in #14033.
